composer/loggers/mlflow_logger.py

```diff
@@ -112,6 +113,7 @@ def init(self, state: State, logger: Logger) -> None:
                     run_name=self.run_name,
                 )
                 self._run_id = new_run.info.run_id
+            mlflow.start_run(run_id=self._run_id)
```

Comment:
nice, yea this (along with setting the `tracking_uri` above) should fix the mlflow fluent api vs mlflow client issues